apps/condo/lang/en/messages/METER_SUBMIT_READINGS_REMINDER_END_PERIOD/default.njk

```diff
@@ -0,0 +1 @@
+Submit your meter readings! Today is the last day to submit meter readings for {{message.meta.data.monthName}}.
```

Comment:
If a message title will be "Submit your meter readings!", then we get message shorter
```suggestion
Today is the last day to submit meter readings for {{message.meta.data.monthName}}.
```
Are these text are compiled by products?

Reply:
yes